src/lib.rs

```diff
@@ -30,10 +30,18 @@ pub fn change_dir() {
 pub fn set_commands(debug: bool, builder: &mut ArgsBuilder, matches: &ArgMatches) {
     let mut commands: Vec<String> = Vec::new();
 
+    // --features are injected at start of every cargo invocation
+    let features = value_t!(matches, "features", String).ok();
+
     // Cargo commands are in front of the rest
     if matches.is_present("cmd:cargo") {
         for cargo in values_t!(matches, "cmd:cargo", String).unwrap_or_else(|e| e.exit()) {
             let mut cmd: String = "cargo ".into();
+            if let Some(features) = features.as_ref() {
```

Comment:
This won't work, as the `--features` flag needs to be _after_ the cargo subcommand

Reply:
Will fix later today. It seems supported subcommands are build, check, doc, run, test, bench, install